### DISCUSSION TITLE
Unstacking files in ReadOnly mode

### DISCUSSION THREAD

**Discussion by patelhiren** [photoprism/photoprism#2111] [Q&A] [answered]
Hello,

I just started using PhotoPrism. I have it running in Docker pointed to my nas in ReadOnly mode. I see some files are stacked and would like to unstack them. The documentation points out how to unstack. But the only option I see is download over there. Is is possible to unstack while in using ReadOnly mode?

**Comment by graciousgrey** [comment, marked as answer]
In general it should be possible. What kind of files do you want to unstack? The unstack button should visible if you expand the information of the file that is not primary.

**Reply by patelhiren** [reply]
I have a few files that are named with IMG_xxxx(n).JPG that I want to unstack. After digging some more I did find out why the Unstack button was missing. 

I had disabled the Edit option under Settings, thinking that I have the files mounted as ReadOnly and would not make sense to have Edit. Once I re-enabled Edit, the Unstack button showed up.

I am not sure if I understood the Edit feature fully yet so this may be expected, but as a new user it confused me why Edit would be needed for Unstack.

**Reply by graciousgrey** [reply]
Good to hear that you could unstack them! You can disable stacking by sequential names for the future within settings (https://docs.photoprism.app/user-guide/settings/library/#stacks).

Read only means that PhotoPrism does not touch your originals directory. 
Some edits you can do in PhotoPrism such as adding metadata or unstacking files will not change files in your originals directory (at least until we write metadata changes back to exif) so that's why there is an additional setting.

Good to know that this is confusing for new users.
